tests/foreman/cli/test_activationkey.py

```diff
@@ -49,1578 +49,1577 @@
-        org = make_org()
-        result = setup_org_for_a_rh_repo(
-            {
-                'product': PRDS['rhel'],
-                'repository-set': REPOSET['rhst7'],
-                'repository': REPOS['rhst7']['name'],
-                'organization-id': org['id'],
-            },
-            force_use_cdn=True,
-        )
-        ak = ActivationKey.info({'id': result['activationkey-id']})
-        setup_org_for_a_custom_repo(
+    :expectedresults: Activation key content override was successful
+
+    :CaseLevel: System
+    """
+    result = setup_org_for_a_custom_repo(
+        {'url': FAKE_0_YUM_REPO, 'organization-id': module_org.id}
+    )
+    content = ActivationKey.product_content(
+        {'id': result['activationkey-id'], 'organization-id': module_org.id}
+    )
+    for override_value in (True, False):
+        ActivationKey.content_override(
             {
-                'url': FAKE_0_YUM_REPO,
-                'organization-id': org['id'],
-                'activationkey-id': result['activationkey-id'],
-                'content-view-id': result['content-view-id'],
-                'lifecycle-environment-id': result['lifecycle-environment-id'],
+                'content-label': content[0]['label'],
+                'id': result['activationkey-id'],
+                'organization-id': module_org.id,
+                'value': int(override_value),
             }
         )
-        subs = Subscription.list({'organization-id': org['id']}, per_page=False)
-        subs_lookup = {s['id']: s for s in subs}
-        with VirtualMachine(distro=DISTRO_RHEL7) as vm:
-            vm.install_katello_ca()
-            vm.register_contenthost(org['label'], activation_key=ak['name'])
-            self.assertTrue(vm.subscribed)
-
-            ak_subs = ActivationKey.subscriptions(
-                {'activation-key': ak['name'], 'organization-id': org['id']}, output_format='json'
-            )
-            self.assertEqual(len(ak_subs), 2)  # one for #rh product, one for custom product
-            for ak_sub in ak_subs:
-                self.assertIn(ak_sub['id'], subs_lookup)
-                self.assertEqual(ak_sub['quantity'], '1')
-                amount = subs_lookup[ak_sub['id']]['quantity']
-                regex = re.compile(f'1 out of {amount}')
-                self.assertRegex(ak_sub['attached'], regex)
+        # Retrieve the product content enabled flag
+        content = ActivationKey.product_content(
+            {'id': result['activationkey-id'], 'organization-id': module_org.id}
+        )
+        assert content[0]['override'] == f'enabled:{int(override_value)}'
+
+
+@pytest.mark.tier2
+def test_positive_remove_user(module_org):
+    """Delete any user who has previously created an activation key
+    and check that activation key still exists
+
+    :id: ba9c4b29-2349-47ea-8081-917de2c17ed2
+
+    :expectedresults: Activation Key can be read
+
+    :BZ: 1291271
+    """
+    password = gen_string('alpha')
```

Comment:
Feels like we should create a fixture for a simple user creation, but not necessary at this time.